nuxeo-core/nuxeo-core/src/main/java/org/nuxeo/ecm/core/trash/AbstractTrashService.java

```diff
@@ -58,6 +58,9 @@
  */
 public abstract class AbstractTrashService implements TrashService {
 
+    // TrashService does not have to expose this
```

Comment:
The comment is not necessary. 

Furthermore, it's a good practice to put fields/constants with `protected` visibility after `public` ones.